core/error.rs

```diff
@@ -91,6 +92,24 @@ pub fn get_custom_error_class(error: &Error) -> Option<&'static str> {
   error.downcast_ref::<CustomError>().map(|e| e.class)
 }
 
+pub fn to_v8_error<'a>(
+  scope: &mut v8::HandleScope<'a>,
+  get_class: GetErrorClassFn,
+  error: &Error,
+) -> v8::Local<'a, v8::Value> {
+  let cb = JsRuntime::state(scope)
+    .borrow()
+    .js_build_custom_error_cb
+    .clone()
+    .expect("Custom error builder must be set");
+  let cb = cb.open(scope);
+  let this = v8::undefined(scope).into();
+  let class = v8::String::new(scope, get_class(error)).unwrap();
+  let message = v8::String::new(scope, &error.to_string()).unwrap();
```

Comment:
Errors can also have the `code` parameter so I guess that should be created here as well (or undefined if not used, reuse `this`) and passed to the call.

Reply:
Added to `to_v8_error()`